sdk/core/src/tasks/sboms/snyk/mod.rs

```diff
@@ -45,7 +45,7 @@ mod tests {
                 Box::new(FileSystemStorageProvider::new(
                     "/tmp/harbor/sboms".to_string(),
                 )),
-                Some(PackageService::new(store.clone())),
+                Some(PackageService::new(store)),
```

Comment:
@DerekStrickland Im guessing pre-commit did this. I know I didnt.

Reply:
Yes it did